src/nobel-prizes/src/meilisearch-client.js

```diff
@@ -1,5 +1,5 @@
-const MEILISEARCH_HOST = "https://demos.meilisearch.com"
-const MEILISEARCH_API_KEY = "L4h8HWpdac31f126b8eebbb23e2d10752e86d57d8a388e6910c2fedfb53542547dd4775c"
+const MEILISEARCH_HOST = "https://ms-a79a247a550f-106.lon.meilisearch.io"
```

Comment:
Why is it being removed from the demo droplet?

Reply:
We are trying to migrate and centralize all the demos to the ms-cloud account Nico created for the demos. We felt that, in the long run, managing the demos in the cloud would be easier due to the upcoming version update capability. For now, this only concerns the demos in this droplet: https://demos.meilisearch.com/. I will share more information about it with you in private. Sorry I haven't informed you about this 😞